README.md

```diff
@@ -18,6 +18,7 @@ Want to go into deep? Please checkout the [documentation](docs).
 ### Install it as a Helm Chart
 
 First, please clone this git repository. Then run command: `make install-chart`
+ - Because the default registry uses `ghcr.io/kubesphere-sigs`, if you want to use `docker.io` as the registry for image pull, you can exec `helm install ks-devops chart/ks-devops --set image.registry=kubespheresig`
```

Comment:
In fact, `ghcr.io` is registry address instead of `ghcr.io/kubesphere-sigs`.

Reply:
Good catch. But the name path of dockerhub doesn't allow `-`. So there's no way to adapt `docker.io/kubesphere-sigs/xxx`. How about the following naming:

```
images:
  # support 'ghcr.io/kubesphere-sigs' or 'kubespheresig'
  repository: ghcr.io/kubesphere-sigs
  controller:
    image: devops-controller
    tag: master
```

Reply:
I can pull images using  links like that docker.io/kubesphere-sigs/xxx  with docker  but i don't know if that's the case for kubernetes too

Reply:
This solution looks good

Reply:
@AbdelouahabMbarki please see here: https://docs.docker.com/docker-id/

Reply:
I see , thanks .

Reply:
If you think we should change the naming? @JohnNiang

Reply:
It would be better to keep the status quo.